src/coincident/search/main.py

```diff
@@ -4,9 +4,11 @@
 from typing import Any
 
 import geopandas as gpd
+import odc.stac
 
 # Used to access formatters
 from pystac_client.item_search import ItemSearch as _ItemSearch
+from shapely.validation import make_valid
```

Comment:
Nice. I didn't know about this function. Currently, the code just checks for ccw ordering https://github.com/uw-cryo/coincident/blob/f366a45eaf34ad472f2cd92c8f0212cb5b146c00/src/coincident/search/main.py#L80-L82 

Out of curiosity, were did you encounter an invalid geometry where this was needed?

Reply:
A lot of flight extent polygons from the NOAA Costal LiDAR catalog are snake-like and non-contiguous which resulted 'invalid' polygons even after a `simplify()` with a tolerance of 0.01 (and I believe some of the really bad flight extent polygons were still invalid after `simplify(0.01).make_valid()`, so I buffered those by a small amount before searching for PCD sites). 
There were also issues with `simplify(0.01)` making holes in some of the NEON flight extent polygons which were created by merging all of the tile extents in the flight, and `make_valid()` fixed some of these (see one of our [Slack threads](https://sheanlab.slack.com/archives/C07H6V8G9T7/p1730711707134919?thread_ts=1729546736.948249&cid=C07H6V8G9T7) on this!). I ended up using a minor `buffer` on these as well for initial PCD site searches rather than `simplify`.

Reply:
Gotcha thanks! Could leave this for a separate PR, but if including, you could add one of these polygons to test (can add as a geojson in a 'tests/data' subfolder).